### ISSUE TITLE
Findbugs complains for @Nullable markers when intern=true is specified

### ISSUE THREAD

**Issue by marcodave** [immutables/immutables#400]
We use `intern=true` in some cases, such as boolean flag type classes.
```java
@Value.Immutable(builder = false, copy = false, intern = true)
@JsonDeserialize(as = ImmutableMyFlag.class)
public interface MyFlag {
}
```

Our build process includes a findbugs check. In those cases, it will complain with this message
```
another must be non-null but is marked as nullable [my.package.ImmutableMyFlag$InternProxy] At ImmutableMyFlag.java:[line 119] NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE
```
Checking the generated class, here is line `ImmutableMyFlag.java:119`
```java
@Override
public boolean equals(@Nullable Object another) {
  return instance.equalTo(((InternProxy) another).instance);
}
```
is it possible to generate the Immutable class with `@Nonnull` annotation?

Edit: we are using Immutables `2.2.10`

**Comment by elucash** [comment]
It was once we removed `@nullable`, but FB complained that we override nullable-parameter method `equals` with non-null parameter. While one of the option would be to add `SuppressFBWarning` to the classpath (and so it would auto appear on Immutables generated classes); we could at the same time change the code with innocent null check like `return another != null && instance...`.
Thank you for reporting this!

**Comment by elucash** [comment]
Added null check, hope this will remove warning, for other cases you might add `SuppressFBWarning` to classpath and generated code will use it
Released in 2.2.11 Thank you
